src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -215,68 +222,77 @@ else if (blockState.getBlock() instanceof BlockHugeMushroom)
         }
         else if (blockState.getBlock() instanceof BlockNetherWart)
         {
-            return Items.nether_wart;
+            return Items.NETHER_WART;
         }
         else if (blockState.getBlock() instanceof BlockPistonExtension)
         {
             // Not really sure what we want to do here...
             return blockState.getValue(BlockPistonExtension.TYPE) == BlockPistonExtension.EnumPistonType.STICKY
-                     ? Item.getItemFromBlock(Blocks.sticky_piston)
-                     : Item.getItemFromBlock(Blocks.piston);
+                     ? Item.getItemFromBlock(Blocks.STICKY_PISTON)
+                     : Item.getItemFromBlock(Blocks.PISTON);
         }
         else if (blockState.getBlock() instanceof BlockRedstoneComparator)
         {
-            return Items.comparator;
+            return Items.COMPARATOR;
         }
         else if (blockState.getBlock() instanceof BlockRedstoneLight)
         {
-            return Item.getItemFromBlock(Blocks.redstone_lamp);
+            return Item.getItemFromBlock(Blocks.REDSTONE_LAMP);
         }
         else if (blockState.getBlock() instanceof BlockRedstoneRepeater)
         {
-            return Items.repeater;
+            return Items.REPEATER;
         }
         else if (blockState.getBlock() instanceof BlockRedstoneTorch)
         {
-            return Item.getItemFromBlock(Blocks.redstone_torch);
+            return Item.getItemFromBlock(Blocks.REDSTONE_TORCH);
         }
         else if (blockState.getBlock() instanceof BlockRedstoneWire)
         {
-            return Items.redstone;
+            return Items.REDSTONE;
         }
         else if (blockState.getBlock() instanceof BlockReed)
         {
-            return Items.reeds;
+            return Items.REEDS;
         }
         else if (blockState.getBlock() instanceof BlockSign)
         {
-            return Items.sign;
+            return Items.SIGN;
         }
         else if (blockState.getBlock() instanceof BlockSkull)
         {
-            return Items.skull;
+            return Items.SKULL;
         }
         else if (blockState.getBlock() instanceof BlockStem)
         {
-            return ((BlockStem) blockState.getBlock()).getSeedItem();
+            ItemStack stack = ((BlockStem) blockState.getBlock()).getItem(null, null, blockState);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'stack' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)